library/core/src/hint.rs

```diff
@@ -120,6 +120,7 @@ pub fn black_box<T>(dummy: T) -> T {
     // more than we want, but it's so far good enough.
 
     // SAFETY: the inline assembly is a no-op.
+    #[cfg(not(miri))] // This is just a hint, so it is fine to skip in Miri.
```

Comment:
now there's no return value anymore ;) You need to apply this flag to the `llvm_asm!` invocation or move the `dummy` out of the `unsafe` block

Reply:
Yeah I just noticed when I build-tested this after pushing.^^ Fixed.